.github/workflows/e2e.yml

```diff
@@ -29,11 +29,8 @@ jobs:
       - name: Generate certificates
         run: make -C e2e/certs certs
 
-      - name: Build test helper containers
-        run: make -C e2e update-helper-images
-
-      - name: Build test container
-        run: make -C e2e update-test-image
+      - name: Build test containers
+        run: make -j4 -C e2e update-helper-images update-test-image
```

Comment:
Is the 4 meant to match the # of CPUs? Could we make this `-j$(nproc)` instead?

Reply:
CPU is not the bottleneck here. 
We want to run all builds in parallel because they are mostly downloading image layers.